include/khiva/dimensionality.h

```diff
@@ -20,6 +20,12 @@ typedef std::pair<float, float> Point;
 
 typedef std::pair<int, int> Segment;
 
+struct VisvalingamSummaryPoint {
+    float x;
+    float y;
+    int64_t area;
+};
+
```

Comment:
I think that it can be in the cpp file as it was previously.

Reply:
I put it there, to have the structs definitions in the header file. However, we can move it to the cpp one